### PR TITLE
Beta3

### PR DESCRIPTION
**Pull request by prathamesh-sonpatki**
r? @schneems

### CODE REVIEW

File: config/environments/production.rb [hunk 1]
```diff
@@ -61,12 +61,17 @@
   # require 'syslog/logger'
   # config.logger = ActiveSupport::TaggedLogging.new(Syslog::Logger.new 'app-name')
 
+  if ENV["RAILS_LOG_TO_STDOUT"].present?
+    config.logger = ActiveSupport::TaggedLogging.new(Logger.new(STDOUT))
+  end
```
Comment (prathamesh-sonpatki) [inline config/environments/production.rb]: We can test this now :)

**Comment by prathamesh-sonpatki** [comment]
@schneems  Good to go?

File: config/environments/production.rb (hunk 1 above)
Comment (schneems) [inline config/environments/production.rb]: It's not deployed yet on heroku. Though we can remove the Rails12 factor gem when it is

**Comment by schneems** [comment]
Looks good, thanks!

**Pull request merged**